tests/rptest/util.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+
+class Scale:
+    KEY = "scale"
```

Comment:
nit: python has a real `Enum` class these days

Reply:
I fiddled with this for 30 minutes today and definitely failed completely to figure out how to use Python enum. If you're ever in there feel free to change, but hopefully with the @property interface we'll never see how Scale works internally :/